configs-stretch/lib/systemd/system/networking.service.wb

```diff
@@ -0,0 +1,22 @@
+[Unit]
```

Comment:
what is this file needed for?

Reply:
https://contactless.atlassian.net/browse/SOFT-469